release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3175,6 +3175,48 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-11"]
+=== RHSA-2024:0642 - {product-title} 4.14.11 bug fix and security update
+
+Issued: 2024-02-07
+
+{product-title} release 4.14.11, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0642[RHSA-2024:0642] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0645[RHBA-2024:0645] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.11 --pullspecs
+----
+
+[id="ocp-4-14-11-features"]
+==== Features
+
+The following feature is included in this z-stream release:
+
+[id="ocp-4-14-11-wherabouts-cron-schedule"]
+===== Enabling configuration of whereabouts cron schedule
+
+* The Whereabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a ConfigMap has enabled the configuration of the whereabouts cron schedule. xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].
+
+[id="ocp-4-14-11-bug-fixes"]
+==== Bug fixes
+
+* Previously, upgrading {product-title} can lead to DNS quieries failing due to upstream returning a payload larger than 512 bytes for non-EDNS queries using CoreDNS 1.10.1. With this release, clusters with a non-compliant upstream will retry with TCP upon overflow errors which will prevent disruption of function when upgrading. (link:https://issues.redhat.com/browse/OCPBUGS-28200[*OCPBUGS-28200*])
+
+* Previously, the node.env file would be overwritten on every restart due to a typo in an environment variable. With this release, edits to the node.env will persist after a restart. (link:https://issues.redhat.com/browse/OCPBUGS-27362[*OCPBUGS-27362*])
+
+* Previously, `container_t` could not access DRI (Direct Rendering Infrastructure) devices. With this release, the policy has been updated so `container_t` can now access DRI devices and GPU devices exposed by a device plugin by default. (link:https://issues.redhat.com/browse/OCPBUGS-27275[*OCPBUGS-27275*])
```

Comment:
I get the use of the parentheses to clarify what's probably more often used as an acronym, but the parentheses should probably be used to introduce the acronym for the full term:
```suggestion
* Previously, `container_t` could not access Direct Rendering Infrastructure (DRI) devices. With this release, the policy has been updated so `container_t` can now access DRI devices and GPU devices exposed by a device plugin by default. (link:https://issues.redhat.com/browse/OCPBUGS-27275[*OCPBUGS-27275*])
```

Reply:
You are correct, thanks!